src/main/kotlin/com/ampnet/headlesscmsservice/config/Converters.kt

```diff
@@ -0,0 +1,37 @@
+package com.ampnet.headlesscmsservice.config
+
+import com.ampnet.headlesscmsservice.enums.Lang
+import com.ampnet.headlesscmsservice.enums.MailType
+import com.ampnet.headlesscmsservice.exception.ErrorCode
+import com.ampnet.headlesscmsservice.exception.InvalidRequestException
+import org.springframework.core.convert.converter.Converter
+import org.springframework.stereotype.Component
+import java.lang.IllegalArgumentException
+
+@Component
+class StringToMailTypeConverter : Converter<String, MailType> {
```

Comment:
If these components are used only in the controller, put the file in the controller package.